keras_mxnet_ci/nightly-buildspec.yml

```diff
@@ -0,0 +1,39 @@
++version: 0.2
++
++phases:
++  install:
++    commands:
++      echo "Checking out master branch";
++      git fetch;
++      git checkout master;
++      echo "Installing MXNet";
++      apt-get update;
++      apt-get install -y curl libgfortran3;
++      pip install --upgrade pip;
++      pip install -U mxnet;
++      apt-get install libhdf5-dev;
++      pip install h5py;
++      pip install -U h5py;
++      pip install -U nose;
++      pip install -U pillow;
++      echo "Installing Tensorflow";
++      pip install -U tensorflow;
++      echo "Installing Theano";
++      pip install -U theano;
++      apt-get install -y openmpi-bin;
++      echo "Installing Keras Test Dependencies";
++      pip install -U pytest;
++      pip install -U pytest-pep8;
++      pip install -U pytest-xdist;
++      pip install -U pytest-cov;
++      pip install -e ./[tests];
++      pip install -U keras_applications;
++      pip install -U keras_preprocessing;
++  build:
++    commands:
++      echo "Running Keras Unit Tests and Integration Tests for all the backends";
++      py.test tests/ --ignore=tests/keras/utils/;tests/keras/wrappers/;
++      py.test tests/keras/utils/ --ignore=tests/keras/wrappers/;
++      pytest tests/keras/wrappers/;
```

Comment:
running twice?

Reply:
I think you changed it to run uitils and wrappers separately when fixing the CI timeout issue. I don't think it running twice.

Reply:
Yeah I remember it. Thanks. I missed to see "--ignore"